README.md

```diff
@@ -117,6 +117,44 @@ You can remove the webcast from the DOM by calling destroy method.
 
 `webcast.destroy();`
 
+#### Embedding a Video
+You can embed a video just like a webcast using an empty div tag.
+
+`<div id="embed"></div>`
+
+Use the embedVideo method to display the video on your web page:
+
+```
+const video = revSdk.embedVideo('#embed', formValues.videoId, {
+	log: true,
+	token,
+	baseUrl: formValues.baseUrl
+});
+```
+
+Use the .on method to watch for events about the video:
+
+```
+['error', 'load', 'videoLoaded', 'volumeChanged', 'captionsChanged', 'seeked', 'playerStatusChanged']
+	.forEach(e => video.on(e, data => {
+		const li = document.createElement('li');
+		li.innerHTML = `${new Date().toLocaleTimeString()} ${e}:${stringifyJson(data)}`;
+		logEl.appendChild(li);
+	}));
+  ```
+
+##### Make the player available globally:
+
+```
+globalThis.vbrickEmbed = currentEmbed;
+```
+
+##### Control the player using external commands:
+
+```
+window.vbrickEmbed.play();
+window.vbrickEmbed.pause();
```

Comment:
dont use `window.`

Reference the same variable defined above, but I would just call it `embed` not `video`

Reply:
ok